jest.setup.js

```diff
@@ -0,0 +1,2 @@
+// polyfill globalThis for older nodes
+globalThis = global;
```

Comment:
I don't think this actually work. Probably `global.globalThis = global;`

Reply:
We are also working on removing all the jest tests at some point in favor of karma tests...

Reply:
We'll need to include the `globalThis` shim for Node 10. While the unit tests may run in Node 10 nothing else would since the lib still references `globalThis` without guards.

Reply:
For Locker code itself we changed `globalThis` for `window` instead of the shim, shouldn't we do the same in both libs? (i.e. change this lib to use `window` or change Locker to use shim)

Reply:
> For Locker code itself we changed `globalThis` for `window` instead of the shim, shouldn't we do the same in both libs? (i.e. change this lib to use `window` or change Locker to use shim)

The `window` object doesn't exist in Node. We also don't want to bolt on a `global.globalThis` to production code since the `global` is a shared object among all-the running Node modules. Babel transforms can handle this and other things too.  See https://node.green/#ES2020-features-globalThis for more on what is and isn't supported in Node 10. It looks like [there's an inconsistency in `ownKeys` traps](https://node.green/#ES2018-misc) in Node 10 too which may need to be considered.

Reply:
Ah! Ok, so the shim would be only for Node, I see. But then, why not say this lib targets Node 12 or 14 and avoid the shim?

Reply:
I think, at least in the next branch, I was careful that only in browser.ts in this lib we use window, while in node we use globalThis. I think we can get away with that for now. Also @manuel-jasso the question was really whether or not @nrobertdehault has to pull this lib directly, or is he getting this bundled up with sandbox pkg?

Reply:
Right now I'm getting this as a transitive dependency of the sandbox pkg.

Reply:
@caridy @nrobertdehault, @locker/sandbox has this lib already bundled, we used to have it as an external dependency, I need to clean this up and move this lib to be a devDependency (i.e. bundled) for @locker/sandbox.